.github/workflows/docker-publish.yml

```diff
@@ -24,8 +24,7 @@ on:
         required: true
 
 env:
-  # github.repository as <account>/<repo>
-  IMAGE_NAME: ${{ github.repository }}
+  IMAGE_NAME: jgosmann/dmarc-metrics-exporter
```

Comment:
Right, in my fork I pushed to a dockerhub handle which didn't match my github account.

Since the image name was hard-coded here https://github.com/jgosmann/dmarc-metrics-exporter/blob/main/.github/workflows/docker-publish.yml#L70 the idea was to give the hard-coded value a prominent location and use the same value for `metadata-action@v4` and `build-push-action@v3`.

Another option would be to configure the `IMAGE_NAME` as github action env variable.

Going back to `IMAGE_NAME: ${{ github.repository }}` seems fine too.

Reply:
> Another option would be to configure the IMAGE_NAME as github action env variable.

That would give the most flexibility, but if one has to configure an env variable, one might as well change the hard-coded value. I don't think I have a strong preference here. 🤷🏻‍♂️

Reply:
Maybe the best solution would be to use `env.IMAGE_NAME` (from action environment) and fallback to `github.repository`. Unfortunately I couldn't figure out how to make this work.